src/main/java/com/exadel/discount/entity/Discount.java

```diff
@@ -16,37 +16,64 @@
 import javax.persistence.JoinTable;
 import javax.persistence.ManyToMany;
 import javax.persistence.Table;
+import java.time.LocalDateTime;
+
 import java.util.Set;
 import java.util.UUID;
 
+@Data
 @Entity
 @Table(name = "discounts")
-@Data
 @NoArgsConstructor
 @AllArgsConstructor
-@EqualsAndHashCode(exclude = {"tags"})
-@ToString(exclude = {"tags"})
+@EqualsAndHashCode(exclude = {"tags", "coupons", "favorites"})
+@ToString(exclude = {"tags", "coupons", "favorites"})
```

Comment:
Hope, U do not mind my edition... )

Reply:
I think adding something that doesn't work to dev is not a good idea. Nikolay's PR doesn't have Coupon and Favorite. It will better to add these fields later with entities that respond for them

Reply:
And I also think it's not good to perform merging dev into not yours branch. As a result we have not working code

Reply:
I agree with Elizaveta, add part code with class, which i don't have, it's very bad idea.
Also about merge dev to my branch, don't make this more.